apiary.apib

```diff
@@ -788,6 +788,7 @@ Many member designs can be created from a single roster design.
 
 To create a member design, the `/roster/` endpoint must recieve valid values for **all**
 roster fields specified for a given roster design.
+You can send empty values (e.g. `""` or `null`), for example to generate a design without a name or number.
```

Comment:
maybe you can continue the sentence above instead say that however, you can send empty values for those those fields if you wish to generate a design without a name of number.

Reply:
Makes sense - but I think it's not important enough to warrant more iterations on this copy right now 🙈